themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers - especially bringing to life Pulumi's Automation API!
+
+### Paul Hicks
+
+<img src="paul-hicks.png" width="200" height="200" alt="Paul Hicks"/>
+<a data-track="github" href="https://github.com/tenwit"><i class="fab fa-github"></i></a>
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
+
+### Ringo De Smet
+
+<img src="ringo-smet.png" width="200" height="200" alt="Ringo De Smet"/>
+<a data-track="twitter" href="https://twitter.com/ringods"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/ringods"><i class="fab fa-github"></i></a>
+
+Ringo is one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/) - a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials. Ringo (along with fellow Puluminary Simen A. W. Olsen) have brought their vision for expanding the Pulumi community to life with the creating of this community-operated place for anyone to communicate, contribute, and collaborate with other members of the Pulumi community and build abstractions and SDKs that can be shared and used by anyone. Ringo also created the [Pulumi Resource Type for Concourse](https://github.com/ringods/pulumi-resource).
+
+### Rizel Scarlett
+
+<img src="rizel-scarlett.png" width="200" height="200" alt="Rizel Scarlett"/>
+<a data-track="twitter" href="https://twitter.com/blackgirlbytes"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/blackgirlbytes"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://www.linkedin.com/in/rizel-bobb-semple/"><i class="fab fa-linkedin"></i></a>
+
+Rizel brings her remarkable talent for guidance and teaching to the Pulumi community. She has participated in many Pulumi events, including [Cloud Engineering Summit](https://www.pulumi.com/resources/from-code-to-cloud/), and [appearing on the PulumiVision livestream](https://youtu.be/2uQEIYuJBZ4).
+
+### Simen A. W. Olsen
+
+<img src="simen-olsen.png" width="200" height="200" alt="Simen W Olsen"/>
+<a data-track="github" href="https://github.com/cobraz"><i class="fab fa-github"></i></a>
+
+Another one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/), Simen has a great passion for technology communities, but especially for the Pulumi community. Simen has also contributed code to the [Pulumi GitHub Action](https://github.com/pulumi/actions)!
```

Comment:
I'd change this to Simen was instrumental on the GitHub Action rewrite - he largely architected that and in his own free time

Reply:
thank you! I didn't have context on what he did for it, and I meant to ask you about that. This is great and I'll update to include this.